cogs/moderator.py

```diff
@@ -75,12 +75,19 @@ async def verify(self, ctx: Context, member):  # parameter only for pretty help.
             member: The member who is to receive a role.
         """
         global verified
-        try:
-            member: Union[Member, User] = ctx.message.mentions[0]
-        except IndexError:
+
+        if ctx.message.mentions:
+            student: Union[Member, User] = ctx.message.mentions[0]
+        elif member:
+            guild: Guild = self.bot.guilds[0]
+            student: Union[Member, User] = guild.get_member(int(member))
```

Comment:
Catch the Value Error that could come here. in that case, raise a `bad_int_argument_handler.py` Error (just noticed that the classname in that file is wrong)